modules/ROOT/pages/Development/Satisfactory/CustomLevels.adoc

```diff
@@ -0,0 +1,66 @@
+= Custom Levels
+
+Custom Levels
+
+== Overview
+
+Satisfactory Update 8 brings with it a host of new features, and one of those features is the ability to create custom levels.
+
+Custom levels depend on many assets, all of which will be detailed below, additionally, an example can be found within the ExampleMod included in the SML project.
+
+The following will explain the different parts of creating a level, for a guide on how to use these to create your own level, skip to xref::/Development/Satisfactory/CustomLevels.adoc#LevelCreation[Creating a level]
+
+== SMLGameMapData
+
+SML implements a custom blueprint class to use as the base of all custom maps, named SMLGameMapData. This defines all the info required for the game to display the level on the main menu, as well as other info, such as what level the game should open. The fields used to define this info are:
+
+* {blank}
++
+Map Asset::
+  A drop down to select the level you created.
+* {blank}
++
+Map Name::
+  A field to define the name of the level as it should show in the main menu.
+* {blank}
++
+Map Description::
+  This field determins the map description as it should show in the main menu.
+* {blank}
++
+Map Icon::
+  Defines the image to show for the map in the main menu.
+* {blank}
++
+Starting Locations::
+  Created as an array, this defines the different possible locations the pioneer can spawn in. Refer to the Defining Spawn Points section on how to use this array.
+
+== Starting Locations Array
+
+The starting locations array, part of the SMLGameMapData asset defines where the player is able to start at.
+
+    TO-DO: Figure out how to create/add Starting Location objects, and create the array

```

Comment:
Please use exactly `TODO` to label todos so that all of them are easily searchable